tools/amend-event-data.js

```diff
@@ -434,6 +434,13 @@ const patches = {
       pattern: { type: 'audioprocess' },
       matched: 1,
       change: { interface: 'AudioProcessingEvent' }
+    },
+    {
+      // https://github.com/WebAudio/web-audio-api/issues/2591
+      // https://github.com/WebAudio/web-audio-api/issues/2590
+      pattern: { type: 'error' },
+      matched: 1,
+      change: { interface: 'Event' }
```

Comment:
I understand https://github.com/WebAudio/web-audio-api/issues/2590 argues that `ErrorEvent` is wrong, but the spec currently says `ErrorEvent` - shouldn't we stick to the spec until this acknowledges by the spec itself?
```suggestion
      change: { interface: 'ErrorEvent' }
```